python/GafferImageUI/ImageViewUI.py

```diff
@@ -237,7 +237,6 @@ class _ColorInspectorPlugValueWidget( GafferUI.PlugValueWidget ) :
 	def __init__( self, plug, **kw ) :
 
 		frame = GafferUI.Frame( borderWidth = 4 )
-		frame._qtWidget().setObjectName( "gafferDarker" )
```

Comment:
This widget appears to have become totally transparent - need to restore the old semi-transparent dark background somehow.

Reply:
Fixed in https://github.com/GafferHQ/gaffer/pull/3229/commits/f956d7a11eb96ec38c163c06ccff6c49790d6acd